src/Returns/Domain/Items/ReturnItem.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Wms\Returns\Domain\Items;
+
+use MyParcelCom\Wms\Returns\Domain\Payment\Currency;
+use MyParcelCom\Wms\Returns\Domain\QuestionAnswer\QuestionAnswer;
+use MyParcelCom\Wms\Returns\Domain\QuestionAnswer\QuestionAnswerCollection;
+
+/**
+ * ReturnItem extends the ArrayObject class with getters to make
+ *  it convenient to extract data from a return coming in from MyParcel.
+ */
+readonly class ReturnItem
+{
+    public function __construct(
+        public string $externalReference,
+        public int $quantity,
+        public int $priceAmount,
+        public Currency $currency,
+        public string $description,
+        public PreferredOutcome $preferredOutcome,
+        public ?WeightUnit $weightUnit = null,
+        public ?int $weight = null,
+        public ?string $sku = null,
+        public ?string $name = null,
+        public ?string $comment = null,
+        public ?string $returnReason = null,
+        public ?string $imageUrl = null,
+        public ?QuestionAnswerCollection $questionAnswers = null,
```

Comment:
might be easier if we make this mandatory because we can pass an empty collection when there are no questionAnswers